### ISSUE TITLE
build error

### ISSUE THREAD

**Issue by pasteleclairs** [carreraSilvio/BrightAnimator#1]
<img width="1439" alt="Screen Shot 2020-11-20 at 11 05 17 AM" src="https://user-images.githubusercontent.com/74750967/99752981-53371a00-2b20-11eb-8212-846fc3ce83e4.png">

Hi, wondering if I did something wrong that caused the build error

**Comment by carreraSilvio** [comment]
Yes! I made a mistake by placing an editor script to go in the runtime assembly. I pushed a fix!

**Comment by dirrogate** [comment]
**edit** Solved by deleting the script from Runtime/Editor folder before building.

Errors again on trying to build, in Unity 2020.2.1f1 @carreraSilvio 
![bright_sound_error](https://user-images.githubusercontent.com/18515826/104428981-8b962a00-559e-11eb-9caf-3a25ea4e6074.JPG)
